manifests/rule.pp

```diff
@@ -1,28 +1,53 @@
 # defined resource which creates a single rule in a specific chain
```

Comment:
we should probably add multiple examples here for all the differences

Reply:
done, i will add some more after I implemented the negate saddr/daddr stuff.